src/test/create-user.test.ts

```diff
@@ -0,0 +1,62 @@
+import { setup } from '../setup';
+import * as request from 'supertest';
+import { expect } from 'chai';
+import { getRepository } from 'typeorm';
+import { User } from '../entity/User';
+
+before(() => {
+  setup();
+});
```

Comment:
Esse script de setup poderia ser rodado apneas uma vez no início dos testes, né? Uma forma de fazer isso é vc rodar um arquivo de testes só, e depois chamar os outros:

```ts
// Exemplo: test/index.ts

before(async () => {
  await setup();
});

require('create-user.test');
require('x.test');
require('y.test');
```